tests/fixtures.py

```diff
@@ -28,13 +29,13 @@
 
 AUTH_HEADER = {
     "X-RH-IDENTITY": "eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6I"
```

Comment:
did this ^ portion of the header remain the same or is this mistakenly missed during the copy paste of the new x-rh-identity?

Reply:
It is the same as it just encodes characters and we have the first 3 lines the same in old and new as well.